farmbot_ext/test/farmbot_ext/amqp/auto_sync_channel_test.exs

```diff
@@ -41,4 +41,7 @@ defmodule FarmbotExt.AMQP.AutoSyncChannelTest do
     allow(MockPreloader, test_pid, pid)
     assert_receive :ding!, wait
   end
+
+  test "setting a channel" do
```

Comment:
if you leave it as `test "setting a channel"`, this test will appear as pending.